packages/core/build/build.package.ts

```diff
@@ -45,7 +44,7 @@ function createPackageFile() {
   // https://nodejs.org/api/packages.html#packages_subpath_exports
   const packageFile = fs.readJsonSync('../dist/core/package.json');
   const packageComponentNames = read('../dist/core')
-    .filter(f => f.endsWith('register.js'))
+    .filter(f => f.endsWith('register.js') && !f.includes('/internal/'))
```

Comment:
The build will add all of the entrypoints to the exports proeprty in the package.json. This enables bundlers and tooling to know what imports are part of the public API. For each component we create a entrypoint `@cds/core/button/register.js` for example. There was a bug where a invalid entry was being added due to a register file in the internal utilities. This filters out the internals and only generates for the components.